options/components/webTopContent/top-contents/WebKeywordSearchBox.jsx

```diff
@@ -0,0 +1,130 @@
+import { faMagnifyingGlass, faRotate } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useContext, useState } from "react";
+
+import { WebSearchContext } from "../../../context/WebSearchContext";
+
+export default function WebKeywordSearchBox({ selectData, urlNewList }) {
+  const [currentSelectValue, setCurrentSelectValue] = useState(
+    selectData[0].selectType
+  );
+
+  return (
+    <div className="flex">
+      <WebSelectBox
+        selectData={selectData}
+        setCurrentSelectValue={setCurrentSelectValue}
+        currentSelectValue={currentSelectValue}
+      />
+      <WebSearchBox urlNewList={urlNewList} />
+    </div>
+  );
+}
+
+function WebSelectBox({
+  selectData,
+  setCurrentSelectValue,
+  currentSelectValue,
+}) {
+  const [isShowOptions, setIsShowOptions] = useState(false);
+
+  const handleOnChangeSelectToggle = () => {
+    return setIsShowOptions(!isShowOptions);
+  };
+
+  const handleOnChangeSelectValue = (e) => {
+    const { innerText } = e.target;
```

Comment:
```suggestion
  const handleOnChangeSelectValue = (event) => {
    const { innerText } = event.target;
```
하단부에 작성해주신 코드와 같이 event 객체를 뜻하는 e를 event로 바꾸면 더 좋을 것 같습니다!

Reply:
받아오는 event 를 전부 event로 표기했다고 생각했었는데 누락되었었네요.
확인 감사합니다. 참고하여 변경하도록 하겠습니다.